### PR TITLE
Eng/pr windows fix

### PR DESCRIPTION
**Pull request by cachemeifyoucan**
Get windows to build with branch.

### CODE REVIEW

**Comment by cachemeifyoucan** [comment]
@swift-ci please test

File: clang/test/CAS/lit.local.cfg [hunk 1]
```diff
@@ -1,3 +1,7 @@
 config.substitutions.append(('%{objroot}', config.clang_obj_root))
 if not config.have_ondisk_cas:
     config.unsupported = True
+
+import platform
+if platform.system() not in ['Linux', 'Darwin']:
```
Comment (benlangmuir) [inline clang/test/CAS/lit.local.cfg]: Maybe this should be `platform.system() == 'Windows'` since I'm guessing it works on other unix-like systems?

Comment (cachemeifyoucan) [reply clang/test/CAS/lit.local.cfg]: Sure. That probably matches better with `LLVM_ON_UNIX` but neither option is an exact match.

**Comment by cachemeifyoucan** [comment]
@swift-ci please test

**Review by benlangmuir — APPROVED** [review]

**Pull request merged**